layouts/index.html

```diff
@@ -0,0 +1,36 @@
+{{ define "main" }}
+<div class="featured-story featured-story-nid-default-featured-story featured-story-light" id="featured-story" data-publish-target="asciidoc">
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-12 col-sm-offset-2 col-md-16 col-md-offset-0 featured-story-block featured-story-block-content match-height-item-by-row featured-container"
+                 style="height: 279px;">
+                {{ range where .Site.Pages "Params.block_type" "purpose" }}
+                {{ .Content }}
+                {{ end }}
+                {{/*
```

Comment:
We should remove content that we have commented, as we have version control for that!

Reply:
Ooops, I didn't push my last commit 😬